app/requests/remote/user.ts

```diff
@@ -105,27 +103,39 @@ export const login = async ({ldapOnly = false, loginId, mfaToken, password}: Log
 };
 
 export const loadMe = async ({deviceToken, user}: LoadMeArgs) => {
-    let currentUser: RawUser = user;
+    let currentUser = user ?? undefined;
 
     const {activeServerDatabase, error} = await getActiveServerDatabase();
     if (!activeServerDatabase) {
-        return {error};
+        return {
```

Comment:
is there a reason why activeDatabase is not defined here ?

Reply:
An extra layer of security.  If we are 100% sure that it is set, then we can remove the check.  We can discuss about it tomorrow.

Reply:
Well if `createAndSetActiveDatabase` fails it should be handled by the above catch, right? no changes needed.. as I said, this whole function will be refactored by me

Reply:
Right.  Thanks .